backend/src/ml_space_lambda/app_configuration/lambda_functions.py

```diff
@@ -0,0 +1,62 @@
+#
```

Comment:
unrelated to this file: will this mean that users will be retrieveing the app configuration every time the page loads? Does this add significant initial load time? Seems unnecessary 

I am mostly wondering why is this not just a config file that gets dropped in the static site bucket every time we have an update in configuration? why the need for an API?

Reply:
Good question! 

If you put it in the website bucket then it will only get pulled when they do a hard refresh. So if admins are making config changes they're not going to be reflected in the browser for any active users.

You also need the config to be readily available for the backend because when users make requests we need to grab the latest config in the backend to make sure they're not making invalid requests. 

We also mitigate this concern by caching the results from `get_configuration` - most requests won't even make it to Lambda, APIGW will see a request for GET and return the cached response it already has.
* we mitigate returning outdated configs by invalidating/clearing the cache when an update is made